server/audit_math/macro.py

```diff
@@ -200,6 +205,9 @@ def get_sample_sizes(
 
     if not U:
         return 1
+    elif U < 0:
+        # This means we have a tie
+        return len(reported_results)
```

Comment:
interesting, so in a tie we do a full recount. i was thinking we'd just fail but i guess this is a more graceful answer

Reply:
Yeah. I don't expect Arlo will ever be used in this way (except for opportunistic contests, maybe?) but regardless it hedges against oddities like the funky CVRs @mcchilders was using.